var/spack/repos/builtin/packages/conduit/package.py

```diff
@@ -85,7 +85,7 @@ class Conduit(Package):
     # CMake
     #######################
     # cmake 3.8.2 or newer
-    depends_on("cmake@3.8.2:3.17.9999", type='build')
+    depends_on("cmake@3.8.2:", type='build')
 
```

Comment:
the 3.17 constraint was added b/c of a recent issue with CMake 3.18.0 and CMake's FindHDF5 logic:
https://github.com/spack/spack/pull/17937

However -- now that spack has cmake 3.18.1 (and 3.18.2), which I belive have a fix for this issue -- I think its safe to again relax this constraint.